src/main/java/com/twingly/search/Query.java

```diff
@@ -0,0 +1,157 @@
+package com.twingly.search;
+
+import com.twingly.search.exception.TwinglyException;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * The type Query.
+ */
+public class Query {
+    private static final String BASE_URL = "http://api.twingly.com";
```

Comment:
OK, I've tried both URL connection client and Apache HTTP client, but received in both situations same exception, here's the stack trace:
```Java
25, 2016 11:36:13 PM co.freeside.betamax.proxy.ProxyServer start
INFO: Betamax proxy is binding to /0.0.0.0:5555
0      2016-02-25 23:36:14,774 WARN  [LittleProxy-ProxyToServerWorker-0] impl.ProxyToServerConnection.exceptionCaught (ProxyToServerConnection.java:347) - (HANDSHAKING) [id: 0xe2cfafe5, 0.0.0.0/0.0.0.0:52634 :> /80.252.171.80:443]: Caught exception on proxy -> web connection
io.netty.handler.codec.DecoderException: javax.net.ssl.SSLException: Received fatal alert: handshake_failure
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:263)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:131)
	at io.netty.channel.DefaultChannelHandlerContext.invokeChannelRead(DefaultChannelHandlerContext.java:337)
	at io.netty.channel.DefaultChannelHandlerContext.fireChannelRead(DefaultChannelHandlerContext.java:323)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:785)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:101)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:485)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:452)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:346)
	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:101)
	at java.lang.Thread.run(Thread.java:745)
Caused by: javax.net.ssl.SSLException: Received fatal alert: handshake_failure
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:208)
	at sun.security.ssl.SSLEngineImpl.fatal(SSLEngineImpl.java:1666)
	at sun.security.ssl.SSLEngineImpl.fatal(SSLEngineImpl.java:1634)
	at sun.security.ssl.SSLEngineImpl.recvAlert(SSLEngineImpl.java:1800)
	at sun.security.ssl.SSLEngineImpl.readRecord(SSLEngineImpl.java:1083)
	at sun.security.ssl.SSLEngineImpl.readNetRecord(SSLEngineImpl.java:907)
	at sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:781)
	at javax.net.ssl.SSLEngine.unwrap(SSLEngine.java:624)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:881)
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:827)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:802)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:232)
	... 10 more
3      2016-02-25 23:36:14,777 WARN  [LittleProxy-ProxyToServerWorker-0] impl.ProxyToServerConnection.exceptionCaught (ProxyToServerConnection.java:350) - (HANDSHAKING) [id: 0xe2cfafe5, 0.0.0.0/0.0.0.0:52634 :> /80.252.171.80:443]: Disconnecting open connection
```